ironmon_tracker/screens/TrackerScreen.lua

```diff
@@ -632,16 +634,15 @@ function TrackerScreen.drawPokemonInfoArea(pokemon)
 			Drawing.drawButton(TrackerScreen.Buttons.PCHealAutoTracking, shadowcolor)
 		end
 	else
-		if Tracker.Data.trainerID ~= nil and pokemon.trainerID ~= nil then
+		-- Assumes we are in Battle, unsure if this check is neccessary
```

Comment:
Good catch and smart habit to avoid assuming anything. I suppose what I was thinking here was eventually we'd extend the route info stuff to work outside of battle, mostly for mapping trainers.